djstripe/models/core.py

```diff
@@ -371,6 +371,7 @@ class Customer(StripeModel):
     expand_fields = ["default_source"]
     stripe_dashboard_item_name = "customers"
 
+    address = JSONField(null=True, blank=True, help_text="The customer’s address.")
```

Comment:
If you've been consistent about using the fancy apostrophe character, go for it, otherwise please use ascii `'` here instead of `’` (same goes for below).